metis-data-checker/metis-data-checker-service/src/main/java/eu/europeana/metis/data/checker/service/persistence/RecordDao.java

```diff
@@ -61,16 +61,15 @@ public RecordDao(AbstractConnectionProvider connectionProvider) throws IndexingE
    * Persist a record in mongo and solr
    *
    * @param rdf The record
-   * @param recordDate
    * @throws IndexingException In case indexing failed.
    */
   public void createRecord(RDF rdf, Date recordDate) throws IndexingException {
```

Comment:
Sonarlint: "document the parameter(s): recordDate"

Reply:
Done